cfn-resources/cluster/cmd/resource/resource.go

```diff
@@ -23,446 +41,261 @@ func castNO64(i *int64) *int {
 	x := cast.ToInt(&i)
 	return &x
 }
+
 func cast64(i *int) *int64 {
 	x := cast.ToInt64(&i)
 	return &x
 }
 
+// validateModel inputs based on the method
+func validateModel(fields []string, model *Model) *handler.ProgressEvent {
+	return validator.ValidateModel(fields, model)
+}
```

Comment:
We can get rid of this validateModel function, and directly use the validator.ValidateModel(fields, model)

Reply:
that should be fine, if any change happens to ValidationModel we can do it in the common place.